src/clr-angular/data/datagrid/providers/page.ts

```diff
@@ -12,6 +12,8 @@ import { StateDebouncer } from './state-debouncer.provider';
 export class Page {
   constructor(private stateDebouncer: StateDebouncer) {}
 
+  public activated = false;
```

Comment:
Since this is public, should it have a test?

Reply:
The property itself is not important to test, but the side effects of it being set. I think the test covers the cases except when changePage is triggered.